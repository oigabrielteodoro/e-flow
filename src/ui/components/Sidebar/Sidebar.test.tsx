import React from 'react'
import { render, screen } from '@testing-library/react'

import { Sidebar } from '.'

describe('<Sidebar />', () => {
  it('should render correctly', () => {
    const { container } = render(<Sidebar />)

    expect(screen.getByAltText('Monetus Logo')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
