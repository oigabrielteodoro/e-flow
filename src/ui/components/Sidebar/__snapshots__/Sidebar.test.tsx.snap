// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> should render correctly 1`] = `
.c0 {
  background: #ffffff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 9.6rem;
  height: 100vh;
}

.c0 nav {
  margin-top: 3rem;
}

.c0 nav ul {
  position: relative;
}

.c0 nav ul li::before {
  content: '';
  position: absolute;
  background: #f06400;
  width: 0.4rem;
  top: 50%;
  height: 4.8rem;
  left: -100%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  border-radius: 0 1.2rem 1.2rem 0;
}

.c1 {
  padding: 2rem 0;
}

<aside
  class="c0"
>
  <img
    alt="Monetus Logo"
    class="c1"
    src="/img/logo.svg"
  />
  <nav>
    <ul>
      <li>
        <img
          alt="Home Icon"
          height="32"
          src="/img/icon-home.svg"
        />
      </li>
    </ul>
  </nav>
</aside>
`;
